src/components/Sections/Lessons/tabs/Series/Container.js

```diff
@@ -96,11 +96,11 @@ const SeriesContainer = ({ t }) => {
 
   const getHeader = node => node.id === "byTopics"
     ? t(`lessons.tabs.seriesTree.byTopics`)
-    : `${t(`lessons.tabs.seriesTree.bySourcesOf`)}${node.name}`;
+    : `${t(`lessons.tabs.seriesTree.bySourcesOf`)}${` `}${node.name}`;
```

Comment:
it is just space, not ${` `}

Reply:
the space is needed. doesn't work other way